src/utils/hooks/useLoadApplication.ts

```diff
@@ -163,6 +163,20 @@ const useLoadApplication = ({ serialNumber, networkFetch }: UseGetApplicationPro
 
     const canApplicantMakeChanges = application.template?.canApplicantMakeChanges ?? true
 
+    const scheduledEvents: ApplicationScheduledEvent[] = (
+      (application?.triggerSchedules?.nodes || []) as {
+        id: number
+        timeScheduled: string
+        eventCode: string
+        isActive: boolean
```

Comment:
Can't just use `as ApplicationScheduledEvent[]` since the incoming data has the date as an ISO string, but I've converted it to a JS Date to put in the structure